gtk4/src/bitset_iter.rs

```diff
@@ -0,0 +1,171 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::Bitset;
+use glib::translate::*;
+
+#[derive(Copy, Clone)]
+pub struct BitsetIter(ffi::GtkBitsetIter);
```

Comment:
```suggestion
pub struct BitsetIter<'a>(ffi::GtkBitsetIter, &'a Bitset);
```

It probably has to borrow the bitset or not? If the bitset disappears the iterator will probably fall apart?

Reply:
```suggestion
pub struct BitsetIter(ffi::GtkBitsetIter, PhantomData<&'a Bitset>);
```

Reply:
Use `PhantomData` if you don't need to actually access the bitset

Reply:
> It probably has to borrow the bitset or not? If the bitset disappears the iterator will probably fall apart? 

aha, I will give it a try and see

Reply:
So we need to keep a reference of the Bitset around otherwise the iterator falls apart. I went ahead and used the PhantomData as you suggested, but that means we can't implement `Iterator` due to this difference
```
    = note: expected fn pointer `fn(&mut BitsetIter<'a>) -> Option<_>`
               found fn pointer `fn(&'a mut BitsetIter<'a>) -> Option<_>`
```
any ideas?

Reply:
Never mind, fixed it :) I have added a test case for that, but it still fails if the bitset is dropped